res/sections/19-Les04042017.tex

```diff
@@ -227,11 +227,14 @@ \subsubsection{IDS Intelligence Systems}
 
 \subsubsection{Honeypot \& Honeynet}
 
-Sostanzialmente sono delle ``trappole''. In un \textit{honeypot} si vuole
-indurre l'attaccante ad entrare in un certo sistema, mentre l'\textit{honeynet}
-è un modo per poter studiare l'attaccante tramite la simulazione di una rete che
-possa rappresentare del valore per l'attaccante e hanno un buon
-funzionamento\footnote{Vengono spesso utilizzate dalle università}.
+Un \textit{honeypot} è un sistema su cui è sono montati software spciali
+che sembrano facile da essere hackerati,
+Mentre un l'\textit{honeynet} è una rete in cui sembra semplice accedere
+e che sembra offrire del valore.   
```

Comment:
Anche gli `honeypot` comunque, non solo le `honeynet`

Reply:
C'è pure un typo. Come consigli di riscrivere?

Reply:
Non saprei. Cancella la parte in cui offrono del valore e mettila alla fine, tipo

```
queste tipologie di sistemi vengono accedute dagli attaccanti in quanto sembrano avere del vero valore al loro interno, ingannandoli.
```